app/views/home/index.html.erb

```diff
@@ -1 +1,17 @@
-<h>Hello World</h>
+<main class="home">
+  <header class="home__header"><a href="#" class="home__logo">ground rules</a></header>
+  <%= render 'svg/home-ill-cup' %>
+  <section class="home__section">
+      <h1>Created for tired people, by slightly less tired people.</h1>
+      <nav class="home__nav">
+        <ul>
+          <li><a href="">Find Your Roast</a></li>
+          <li><a href="">Master Your Machine</a></li>
+        </ul>
+      </nav>
```

Comment:
I don't think you need the list in here.  Just a `<nav>` with some `<a>`(s) in it is fine and would read a little better to a screen reader (which announces how many links are in a nav).